README.md

```diff
@@ -4,20 +4,31 @@ This repository contains all models and worlds required by the JdeRobot exercise
 
 The `kinetic-devel` branch has been released on the official ROS build farm and can be downloaded using the command:
 
+## Installation
+
+### ROS Kinetic
+
 ```bash
 sudo apt-get install ros-kinetic-jderobot-assets
 ```
 
+### ROS Melodic
+
+```bash
+sudo apt-get install ros-melodic-jderobot-assets
+```
+
+
 ## ROS Distro Support
 
-|         | Kinetic |
-|:-------:|:-------:|
-| Branch  | [`kinetic-devel`](https://github.com/JdeRobot/assets/tree/kinetic-devel) |
-| Status  |  supported |
-| Version | [version](http://repositories.ros.org/status_page/ros_kinetic_default.html?q=jderobot_assets) |
+|         | Kinetic | Melodic |
+|:-------:|:-------:|:-------:|
+| Branch  | [`kinetic-devel`](https://github.com/JdeRobot/assets/tree/kinetic-devel) |[`master`](https://github.com/JdeRobot/assets/tree/master)|
+| Status  |  supported | supported |
+| Version | [version](http://repositories.ros.org/status_page/ros_kinetic_default.html?q=jderobot_assets) |[version](http://repositories.ros.org/status_page/ros_melodic_default.html?q=jderobot_assets)|
 
 ## ROS Buildfarm
 
-|         |  Kinetic Source  |  Kinetic Debian |
-|:-------:|:-------------------:|:-------------------:|
-| jderobot_assets | [![released](http://build.ros.org/buildStatus/icon?job=Ksrc_uX__jderobot_assets__ubuntu_xenial__source)](http://build.ros.org/view/Ksrc_uX/job/Ksrc_uX__jderobot_assets__ubuntu_xenial__source/) | [![released](http://build.ros.org/buildStatus/icon?job=Kbin_uX64__jderobot_assets__ubuntu_xenial_amd64__binary)](http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__jderobot_assets__ubuntu_xenial_amd64__binary/) |
+|         |  Kinetic Source  |  Kinetic Debian |  Melodic Source  |  Melodic Debian |
```

Comment:
Since there are multiple builds that we should keep a track of, they should all be organized neatly in a table here. I see you have added the builds for Melodic src and deb, this is good, however, it would be better to have all of them (14 in total listed here: http://build.ros.org/search/?q=assets&Jenkins-Crumb=99a80b2cb06504872692e2b82369531b2602da69c8d612edb05a65c893daa884). How about 2 tables, for Ubuntu and Debian, with the ROS version and distro on the y axis and the types of builds on the x axis.